dwitter/static/js/scrolling.js

```diff
@@ -43,6 +43,11 @@ window.onload = function() {
               $(this).off("click.a keyup.a").select();
             });
           });
+
+          moment.locale(navigator.userLanguage || navigator.language || 'en-US');
+          var timestamp = $(div).find('.dweet-timestamp');
+          var postedDate = moment.utc(timestamp.text());
+          timestamp.text(postedDate.local().format('lll'));
```

Comment:
This code is mostly duplicated from `feed.js`. At some point, I think that file and `scrolling.js` should be refactored. Since we don't have any tests covering those files, though, I'm leery of doing that refactor right now.